setup.md

```diff
@@ -1,8 +1,24 @@
-How to set up code for development
-- git checkout https://github.com/bpython/curtsies.git
-- cd curtsies
-- pip install -e .
-
-How to run the tests
-- pip install pyte coverage mock nose
-- nosetests .
\ No newline at end of file
+## Set up for development
+To set up a local repository to begin using curtsies as a developer, run
+
+    $ git clone https://github.com/bpython/curtsies.git
+    $ cd curtsies
+    $ pip install -e .
+
+## Running Tests
+--------
+Tests are written using the unittest framework and are run using using [pyte](https://pypi.org/project/pyte/), [coverage](https://coverage.readthedocs.io/en/coverage-5.1/),  [mock](https://pypi.org/project/mock/), and [nosetests](https://nose.readthedocs.io/en/latest/). To run all tests, do:
```

Comment:
I'd say that pyte and mock are just libraries we use, "testing dependencies," the only thing to mention and link here is nosetests. (btw there's an issue to stop using nosetests, I don't remember why, if you're in a testing mood)